src/homework_oop_v2/Truck.java

```diff
@@ -0,0 +1,20 @@
+package homework_oop_v2;
+
+public class Truck extends Transport implements Serviceable {
+
+    public Truck(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+    }
+
+    public void service() {
+        if (this != null) {
+            System.out.println("Обслуживаем " + this.getModelName());
+            for (int i = 0; i < this.getWheelsCount(); i++) {
+                System.out.println("Меняем покрышку");
```

Comment:
```suggestion
                System.out.println("Меняем покрышку у грузовика");
```
чтобы показать, что реализация другая, чтобы был виден полиморфизм